packages/block-editor/src/components/block-list/style.scss

```diff
@@ -556,7 +556,8 @@
 
 // Position mover arrows for both toolbars.
 .block-editor-block-contextual-toolbar,
-.edit-post-header-toolbar__block-toolbar {
+.edit-post-header-toolbar__block-toolbar,
+.edit-site-header-toolbar__block-toolbar {
```

Comment:
Not something for this PR, I can follow up, but I just realised this is where we probably also need `edit-widgets-header__block-toolbar` and `edit-navigation-layout__block-toolbar`. Hopefully that would solve some of the issues that have been seen.

It doesn't seem completely sustainable to style the toolbar using these classnames. It ties in with an issue I made - https://github.com/WordPress/gutenberg/issues/29965. But not something to worry about right now.

Reply:
Thanks so much for the review.

I profoundly agree it's not sustainable. I forgive the current situation because the dedicated site editor as a separate menu item was created in order to experiment. But as we move forward and if we decide to keep that separate menu item, we need to find a better way to do that. This separate PR I worked on is an example of that: https://github.com/WordPress/gutenberg/pull/30234 — the markup deviates quite a bit, and it's painful to keep in sync; features or bugfixes for one editor built for one are likely to be missing from the other due to this mix.

Reply:
Oh sorry, I just merged with the big green button — let me follow up with a PR with your suggestion.

Reply:
Followup: https://github.com/WordPress/gutenberg/pull/30284